### ISSUE TITLE
Can't figure out how to make responsive

### ISSUE THREAD

**Issue by jwhitty1** [xiaolin/react-image-gallery#502]
The actual images in the gallery keep their original dimensions/resolution.
I need them to conform to a set width/height at different media query levels, but I cannot navigate the css code for the life of me. Everything looks the same.
Any advice on how this can be done? I've been using react for only 2 months so I'm very new to this and while I've found a way to resize the slider part of the gallery ONLY FOR THE FIRST IMAGE SHOWN (it still displaces everything around it when I scroll to the next, taller image)...that isn't actually a fix for this issue.

Your demo shows the images all shrink down at the same dimensions: how?! :(

**Comment by citizen-dror** [comment]
maybe this will help you ..

App.css:

.image-gallery-slide img {
  width: 100%;
  height: 65vh;
}

or

.image-gallery-slide img {
  width: 100%;
  height: 500px;
}

**Comment by jwhitty1** [comment]
thank you, that did help with some things. im struggling to deal with the
way that there is "image-gallery- * " and it goes on and on
-slide
-slides
-slide-this
-slide-this-that...

and im putting the entire thing into a grid (airbnb clone, so you can check
what the search results on airbnb look like for an idea of what im doing)
and somehow there is a huge space above the actual image slider always... i
cant seem to make it go away so that the image slider is at the TOP of the
container and the arrows stay in the center of the image slider

**Comment by xiaolin** [comment]
Can you show a screenshot of the issue? I feel like you're not properly including the CSS but want to make sure.

**Comment by jwhitty1** [comment]
Just a moment yes you're definitely right but my problem is that I can't follow the css (I've only been at this for two months) enough to know what's doing what

**Comment by jwhitty1** [comment]
sorry got a bit carried away with things: here are two screens in firefox
(im testing cross-browser responsiveness atm)

as you can see below there are two different CSS properties with the same
space and I find this really confusing and hard to follow and i have to
just guess what is causing this to move to the left now, versus chrome
where its centered


[image: Screen Shot 2020-04-26 at 1.29.28 AM.png]

[image: Screen Shot 2020-04-26 at 1.27.27 AM.png]
[image: Screen Shot 2020-04-26 at 1.27.31 AM.png]

**Comment by xiaolin** [comment]
Where you able to fix your issue? I realize none of the attached screenshots came through because you responded via email.
